bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/VerisureSession.java

```diff
@@ -86,24 +87,31 @@
     private int apiServerInUseIndex = 0;
     private int numberOfEvents = 15;
     private static final String USER_NAME = "username";
-    private static final String PASSWORD_NAME = "vid";
+    private static final String VID = "vid";
+    private static final String VS_STEPUP = "vs-stepup";
+    private static final String VS_ACCESS = "vs-access";
     private String apiServerInUse = APISERVERLIST.get(apiServerInUseIndex);
     private String authstring = "";
     private @Nullable String csrf;
     private @Nullable String pinCode;
     private HttpClient httpClient;
     private @Nullable String userName = "";
-    private @Nullable String password = "";
+    private @Nullable String passWord = "";
```

Comment:
`password` was correct.
To be changed everywhere

Reply:
Fixed!